client/components/header-nav/README.md

```diff
@@ -0,0 +1,4 @@
+Header Nav
```

Comment:
We already have a `SectionNav` :/ How about `SubMasterBarNav`? ;) I'm sure we've had this discussion on Slack a year ago!